modules/platform/nuxeo-thumbnail/src/main/java/org/nuxeo/ecm/platform/thumbnail/factories/ThumbnailDocumentFactory.java

```diff
@@ -95,7 +95,7 @@ public Blob computeThumbnail(DocumentModel doc, CoreSession session) {
                 }
             }
         } catch (NuxeoException e) {
-            log.warn("Cannot compute document thumbnail", e);
+            log.warn("Cannot compute document thumbnail for: " + doc.getId(), e);
```

Comment:
`log.warn("Cannot compute document thumbnail for: {}", doc:getId, e);`

Reply:
yeah but no, the class is still using log4j 1